unlock-app/src/components/interface/connect/UnlockAccount.tsx

```diff
@@ -264,7 +264,17 @@ export const ConnectUnlockAccount = ({ onExit }: Props) => {
           }}
         />
       )}
-      <div className="grid p-6">
+      <div className="grid gap-4 p-6">
```

Comment:
i am confused about this?

Reply:
Hides the I want to get a wallet button when user choose to sign up for an unlock account but will remove this and keep it consistent on both pages.